chore(Theme): rename open-banking theme (#1327)
dnbexperience/eufemia: packages/dnb-design-system-portal/src/docs/uilib/usage/customisation/theming.md

Contributions are welcome. Heres the [source code](https://github.com/dnbexperience/eufemia-theme-manager).

## Component theming

By default, all the HTML Elements (components) are built by separating the "visual styling" parts from the "functional layout" parts. This way we can create new custom visual styles:

```js
/button/style/_button.scss // layout styles
/button/style/themes/dnb-button-theme-ui.scss // main theme styles
/button/style/themes/dnb-button-theme-eiendom.scss// additional theme styles
```

- The main theme (`ui`) does contain mainly colorization and sizes.

> - While all the raw positionling and layout related properties are in the main `.scss` file, starting with an underscore: `_button.scss`

Reviewer: ```suggestion
- While all the raw positioning and layout related properties are in the main `.scss` file, starting with an underscore: `_button.scss`
```

> Its still possible to overwrite the _layout_ properties to customize our theme even further, it thats is needed.

Reviewer: ```suggestion
It's still possible to overwrite the _layout_ properties to customize our theme even further, if that is needed.
```
